appinfo/info.xml

```diff
@@ -4,7 +4,7 @@
 	<id>firstrunwizard</id>
 	<name>First run wizard</name>
 	<summary>A welcome and introduction wizard for Nextcloud</summary>
-	<description>This application enables a pop-up window when a user first logs into Nextcloud. This window welcomes new users to Nextcloud, and contains links to the standard desktop client, Android app, and the iOS app. The window also shows a link for accessing Nextcloud via WebDAV. At any time, the first run wizard button can be opened again via the link on the personal page.
+	<description>This application enables a pop-up window when a user first logs into Nextcloud. This window welcomes new users to Nextcloud, and contains links to the standard desktop client, Android app, and the iOS app. The window also shows a link for accessing Nextcloud via WebDAV. At any time, the first run wizard can be opened again via the link on the personal page.
```

Comment:
We moved the button to the settings menu (About) 
![image](https://user-images.githubusercontent.com/3404133/48340648-68445d00-e66b-11e8-81a0-9540766b5dd5.png)

Mind to adjust this as well @rakekniven ?

Reply:
Finally done.